addon/-private/execution_context/acceptance-native-dom.js

```diff
@@ -0,0 +1,23 @@
+import ExecutionContext from './native-dom-context';
+
+import {
+  visit
+} from 'ember-native-dom-helpers';
+
+export default function AcceptanceExecutionContext(pageObjectNode) {
+  ExecutionContext.call(this, pageObjectNode);
+}
+
+AcceptanceExecutionContext.prototype = Object.create(ExecutionContext.prototype);
+
+AcceptanceExecutionContext.prototype.visit = function() {
+  return visit(...arguments);
+};
+
+AcceptanceExecutionContext.prototype.runAsync = function(cb) {
+  window.wait().then(() => {
```

Comment:
This appears to be the last dependency on a global test helper, which means we still have to use `setContext` to be able to use PageObjects in integration tests. We can do the same thing `ember-native-dom-helpers` does here to make this completely independent:

```js
import wait from 'ember-test-helpers/wait';

(window.wait || wait)().then(() => {});
```

Reply:
You are right , seems like it should be fixed.